Cargo.toml

```diff
@@ -113,23 +114,23 @@ uv-requirements-txt = { git = "https://github.com/astral-sh/uv", tag = "0.5.29"
 
 wax = "0.6.0"
 which = "7.0.1"
-
 # Rattler crates
-file_url = "0.2.2"
-rattler = { version = "0.32.0", default-features = false }
-rattler_cache = { version = "0.3.9", default-features = false }
-rattler_conda_types = { version = "0.30.3", default-features = false, features = [
+file_url = { default-features = false, path = "../rattler/crates/file_url" }
+rattler = { default-features = false, path = "../rattler/crates/rattler" }
+rattler_cache = { default-features = false, path = "../rattler/crates/rattler_cache" }
+rattler_conda_types = { default-features = false, features = [
   "rayon",
-] }
-rattler_digest = { version = "1.0.6", default-features = false }
-rattler_lock = { version = "0.22.43", default-features = false }
-rattler_networking = { version = "0.22.4", default-features = false, features = [
-  "google-cloud-auth",
-] }
-rattler_repodata_gateway = { version = "0.21.37", default-features = false }
-rattler_shell = { version = "0.22.19", default-features = false }
-rattler_solve = { version = "1.3.8", default-features = false }
-rattler_virtual_packages = { version = "2.0.3", default-features = false }
+], path = "../rattler/crates/rattler_conda_types" }
+rattler_digest = { default-features = false, path = "../rattler/crates/rattler_digest" }
+rattler_lock = { default-features = false, path = "../rattler/crates/rattler_lock" }
+rattler_menuinst = { default-features = false, path = "../rattler/crates/rattler_menuinst" }
+rattler_networking = { default-features = false, features = [
+  "gcs",
+], path = "../rattler/crates/rattler_networking" }
+rattler_repodata_gateway = { default-features = false, path = "../rattler/crates/rattler_repodata_gateway" }
+rattler_shell = { default-features = false, path = "../rattler/crates/rattler_shell" }
+rattler_solve = { default-features = false, path = "../rattler/crates/rattler_solve" }
+rattler_virtual_packages = { default-features = false, path = "../rattler/crates/rattler_virtual_packages" }
```

Comment:
We can move this to git main

Reply:
Please do so, that would help the CI run